client/end.py

```diff
@@ -0,0 +1,28 @@
+from definitions import *
+import pygame as pg
+
+from buttonPress import Button
+
+def endScreen(screen):
+    button_rejoin = Button(90, 30, "Rejoin", (50,130))
+    back_to_login = Button(200, 30, "Return to Login Screen", (50,180))
+    
+    done = False
+    rejoin = False
```

Comment:
since rejoin is treated as a constant here, please rename it to all caps so it is easy to tell that it is a constant value